MoesifWorker.js

```diff
@@ -448,6 +495,13 @@ async function logRequest(event) {
     return;
   }
 
+  var randomNumber = Math.random() * 100;
+  // Fetch app Config first time or if not fetched within last 5 mins
+  if (!isAppConfigFetched || lastUpdatedAppConfigTime == null || new Date().getTime() > lastUpdatedAppConfigTime + fetchAppConfigTimeDeltaInMins ) {
+    lastUpdatedAppConfigTime = new Date().getTime();
+    event.waitUntil(fetchAppConfig());
```

Comment:
does event need to wait for config to be fetched? Or is there a way to do this asycnhronously? like in await , and not wait for await?

Reply:
It doesn't block the response - https://developers.cloudflare.com/workers/learning/fetch-event-lifecycle#waituntil